vcbuild.bat

```diff
@@ -502,6 +502,9 @@ if %errorlevel% equ 0 goto exit
 echo %1 | findstr /r /c:"test\\addons\\[0-9].*_.*\.cc"
 if %errorlevel% equ 0 goto exit
 
+echo %1 | findstr /c:"test\\addons-napi\common.h"
```

Comment:
Shouldn't that be `"test\\addons-napi\\common.h"`?  (Note the second double slash.)

Reply:
Oh yes I should, sorry missed that. I'll update. Thanks